### PR TITLE
Update document for change log

### PR DESCRIPTION
**Pull request by Juliehzl**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

File: doc/authoring_command_modules/README.md [hunk 1]
```diff
@@ -212,7 +212,6 @@ e.g.:
 Release History
 ===============
 
-0.0.3
```
Comment (MyronFanQiu) [inline doc/authoring_command_modules/README.md]: any story?

Comment (Juliehzl) [reply doc/authoring_command_modules/README.md]: Customers do not need to add version number for next release. We will add it during release.

File: doc/authoring_command_modules/README.md [hunk 2]
```diff
@@ -212,7 +212,6 @@ e.g.:
...
 +++++
```
Comment (MyronFanQiu) [inline doc/authoring_command_modules/README.md]: who does need to add this line?

File: doc/authoring_command_modules/README.md (hunk 1 above)
Comment (MyronFanQiu) [reply doc/authoring_command_modules/README.md]: Sound reasonable. I prefer adding the next release info into history when we finish the current one.

File: doc/authoring_command_modules/README.md (hunk 2 above)
Comment (MyronFanQiu) [reply doc/authoring_command_modules/README.md]: +++++

**Comment by MyronFanQiu** [comment]
@yugangw-msft I prefer we have something like
```
History notes for next release
+++++++++++++++++++
**<command module>**

* This is my customer-facing change.

```
Then during release, we can change the release version to the right one. After release, we should provide the same content in the history.rst.

**Comment by Juliehzl** [comment]
Because in document we have already declare how to add release notes for next release, the part I changed is just an sample to specify where users can add their release notes. I think we can just remove  ```version number``` and ```+++++++++++++++++++```,  and users can know how to add their release notes in the right position with document and sample shown.

**Comment by yugangw-msft** [comment]
The goals is to make sure the sample here is as closer to the [real one](https://github.com/Azure/azure-cli/blob/dev/src/azure-cli/HISTORY.rst) as possible, so users can easily find the spot to input the change log correctly.

**Comment by yugangw-msft** [comment]
Please do rebase 
```
git checkout dev
git pull upstream dev
git checkout changelog
git rebase dev
```

**Comment by yonzhan** [comment]
Since this PR was very old. Let's discuss offline.

**Pull request closed without merging**